<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_For_Add_Service</class>
 <widget class="QWidget" name="Form_For_Add_Service">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добвление услуги</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>НАИМЕНОВАНИЕ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_service">
       <property name="styleSheet">
        <string notr="true">
border-radius : 4px;
border: 1px solid;
border-color: rgb(11, 11, 11);
height:20;
width:140;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ЦЕНА</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="cost_service">
       <property name="styleSheet">
        <string notr="true">
border-radius : 4px;
border: 1px solid;
border-color: rgb(11, 11, 11);
height:20;
width:50;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);</string>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
border-radius : 4px;
border: 1px solid;
border-color: rgb(11, 11, 11);
height:20;
width:100;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
QPushButton::pressed {
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(251, 252, 255), stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
border-radius : 4px;
border: 1px solid;
border-color: rgb(11, 11, 11);
height:20;
width:100;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
QPushButton::pressed {
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(251, 252, 255), stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
